components/Editor/index.tsx

```diff
@@ -0,0 +1,230 @@
+          <DmpIcon icon="table" />
+        </Button>
+        <Popover>
+          <Menu>
+            <MenuItem onAction={() => commands.createTable()}>Insert Table</MenuItem>
+            <Separator />
+            <MenuItem onAction={() => commands.addTableColumnBefore()}>Add Column Left</MenuItem>
+            <MenuItem onAction={() => commands.addTableColumnAfter()}>Add Column Right</MenuItem>
+            <MenuItem onAction={() => commands.deleteTableColumn()}>Delete Column</MenuItem>
+            <Separator />
+            <MenuItem onAction={() => commands.addTableRowBefore()}>Add Row Above</MenuItem>
+            <MenuItem onAction={() => commands.addTableRowAfter()}>Add Row Below</MenuItem>
+            <MenuItem onAction={() => commands.deleteTableRow()}>Delete Row</MenuItem>
+            <Separator />
+            <MenuItem onAction={() => commands.deleteTable()}>Delete Table</MenuItem>
+          </Menu>
+        </Popover>
+      </MenuTrigger>
+    </Group>
+  )
+}
+
+
+const EditorToolbar = () => {
+  const chain = useChainedCommands();
+  const active = useActive();
+
+  return (
+    <Toolbar aria-label="Editor Tools">
+      <Group aria-label="Style">
+        <ToggleButton
+          aria-label="Bold"
+          onPress={() => {
+            chain
+              .toggleBold()
+              .focus()
+              .run();
+          }}
+          isSelected={active.bold()}
+        >
+          <DmpIcon icon="format_bold" />
+        </ToggleButton>
+        <ToggleButton
+          aria-label="Italic"
+          onPress={() => {
+            chain
+              .toggleItalic()
+              .focus()
+              .run();
+          }}
+          isSelected={active.italic()}
+        >
+          <DmpIcon icon="format_italic" />
+        </ToggleButton>
+        <ToggleButton
+          aria-label="Underline"
+          onPress={() => {
+            chain
+              .toggleUnderline()
+              .focus()
+              .run();
+          }}
+          isSelected={active.underline()}
+        >
+          <DmpIcon icon="format_underlined" />
+        </ToggleButton>
+      </Group>
+
+      <Separator orientation="vertical" />
+
+      <Group aria-label="Lists">
+        <ToggleButton
+          aria-label="Bullet List"
+          onPress={() => {
+            chain
+              .toggleBulletList()
+              .focus()
+              .run();
+          }}
+          isSelected={active.bulletList()}
+        >
+          <DmpIcon icon="format_list_bulleted" />
+        </ToggleButton>
+
+        <ToggleButton
+          aria-label="Number List"
+          onPress={() => {
+            chain
+              .toggleOrderedList()
+              .focus()
+              .run();
+          }}
+          isSelected={active.orderedList()}
+        >
+          <DmpIcon icon="format_list_numbered" />
+        </ToggleButton>
+      </Group>
+
+      <Separator orientation="vertical" />
+
+      <TableGroup />
+
```

Comment:
It looks like the AnnotationButton is missing

Reply:
Yea, I don't see it either.

Reply:
Sorry, I thought we weren't doing those but rather will be building comments as a thing related to whole questions, for now, and only adding annotations later on when we know how we will reference them in the database. So to keep things neat I just removed the button for now. Apologies for the misunderstanding!

I can put it back in if you need me to?

Reply:
@andrewebdev  - I see. Yes, so this Rich Text Editor can be used for the comments at a question level, whereas the annotation is very different in that it will be associated with inline comments. As long as we can use the Rich Text Editor example for the same commenting functionality as one our existing site, as our MVP, I think that's ok.